yarn.lock

```diff
@@ -1821,15 +1830,16 @@ __metadata:
   languageName: unknown
   linkType: soft
 
-"ardrive-core-js@npm:2.0.0-alpha-0":
-  version: 2.0.0-alpha-0
-  resolution: "ardrive-core-js@npm:2.0.0-alpha-0"
+"ardrive-core-js@npm:2.0.0-alpha-1":
+  version: 2.0.0-alpha-1
+  resolution: "ardrive-core-js@npm:2.0.0-alpha-1"
   dependencies:
     "@alexsasharegan/simple-cache": ^3.3.3
     arbundles: ^0.6.19
     arweave: ^1.11.4
     arweave-mnemonic-keys: ^0.0.9
     axios: ^0.21.1
+    axios-retry: ^3.6.0
```

Comment:
i've started to favor retry-axios over axios-retry. it gives some more granular control and works in esm modules 

https://www.npmjs.com/package/retry-axios